tasks/vagrant.rb

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env ruby
+require 'json'
+require 'net/http'
+require 'yaml'
+require 'puppet_litmus'
+require 'fileutils'
+
+@supported_platforms = {
+  'ubuntu14.04' => 'ubuntu/trusty64',
+}
+
+def run_local_command(command, wd = Dir.pwd)
```

Comment:
```suggestion
def run_local_command(command, wd = Dir.pwd)
```
should this become a helper function for the module, it is used in docker.